mungegithub/mungers/submit-queue.go

```diff
@@ -545,6 +547,7 @@ func (sq *SubmitQueue) EachLoop() error {
 // RegisterOptions registers options used by the SubmitQueue.
 func (sq *SubmitQueue) RegisterOptions(opts *options.Options) {
 	opts.RegisterStringSlice(&sq.NonBlockingJobNames, "nonblocking-jenkins-jobs", []string{}, "Comma separated list of jobs that don't block merges, but will have status reported and issues filed.")
+	opts.RegisterStringSlice(&sq.PreSubmitJobNames, "presubmit-jenkins-jobs", []string{}, "Comma separated list of jobs that run before merging code.")
```

Comment:
I think so, however I wanted to keep them consistent. I'm not sure if updating the existing flags should be in the scope of this PR or not.